modules/appeals_api/app/swagger/appeals_api/v2/api_description.md

```diff
@@ -43,39 +43,36 @@ The **evidence upload** statuses available for simulation are the statuses liste
-
-1. Use the POST endpoint [`/notice_of_disagreements/evidence_submissions`] to return a JSON service response with the attributes listed below.
-    - `guid`: An identifier used for subsequent evidence upload status requests (not to be confused with the NOD submission GUID)
-    - `location`: A URL to which the actual document package payload can be submitted in the next step. The URL is specific to this upload request, and should not be re-used for subsequent uploads. The URL is valid for 900 seconds (15 minutes) from the time of this response. If the location is not used within 15 minutes, the GUID will expire. Once expired, status checks on the GUID will return a status of `expired`.
-
-1. Client Request: PUT to the location URL returned in step 1.
-    - Request body should be encoded as binary multipart/form-data, equivalent to that generated by an HTML form submission or using “curl -F…”.
-    - No `apikey` authorization header is required for this request, as authorization is embedded in the signed location URL.
-    - The metadata.json file uploaded to the location URL with the evidence documents MUST set the `businessLine` to “BVA” to ensure your supporting  documents route to the Board of Veterans Appeals (BVA).
-    - The JSON key for the metadata.json file is "metadata", the initial file is "content".  Any subsequent file will be "attachment1", "attachment2", and so forth.
-
-1. The service response will include:
-    - HTTP status to indicate whether the evidence document upload was successful.
-    - ETag header containing an MD5 hash of the submitted payload. This can be compared to the submitted payload to ensure data integrity of the upload.
-
-Example `metadata.json` file:
-
-```
-{
-  "veteranFirstName": "Jane",
-  "veteranLastName": "Doe",
-  "fileNumber": "012345678",
-  "zipCode": "94402",
-  "source": "Vets.gov",
-  "docType": "316",
-  "businessLine": "BVA"
-}
-```
-
-You may check the status of your evidence document upload by using GET `/notice_of_disagreements/evidence_submissions/{uuid}`. If, after you've uploaded a document, the status hasn't changed to `uploaded` before 15 minutes has elapsed, we recommend retrying the submission to make sure the document properly reaches our servers.
-
-Evidence may only be uploaded within 90 days of the NOD reaching submitted status. After 90 days an error will be returned if evidence uploads related to this NOD are attempted.
+Our NOD and Supplemental Claim evidence submission endpoints allow a client to upload a document package (documents and metadata) of supporting evidence for their submission by following these steps. 
+
+1. Use the POST endpoint `/notice_of_disagreements/evidence_submissions` or `/supplemental_claims/evidence_submissions` to return a JSON service response with the attributes listed below.
+
+guid: An identifier used for subsequent evidence upload status requests (not to be confused with the NOD or SC submission GUID)
```

Comment:
Shouldn't this be a bullet within the counter? Right now it's broken away from its parent.